src/onepasswordconnectsdk/client.py

```diff
@@ -494,6 +495,9 @@ def new_client_from_environment(url: str):
     """
     token = os.environ.get(ENV_SERVICE_ACCOUNT_JWT_VARIABLE)
 
+    if url is None:
+        url = os.environ.get(CONNECT_HOST_ENV_VARIABLE)
+
```

Comment:
In case `url` is `None`, and the `OP_CONNECT_HOST` environment variable has not been set, shall we throw an exception? Maybe we could have a custom exception, the same as we have for the token, on line 502. What do you think?

Reply:
yes good call! :+1: